controlpanel/api/models/user.py

```diff
@@ -83,10 +83,28 @@ def reset_mfa(self):
         auth0.ManagementAPI().reset_mfa(self.auth0_id)
 
     def save(self, *args, **kwargs):
+        request_user = None
+        request = CrequestMiddleware.get_request()
+        if request:
+            request_user = request.user
+
         try:
-            User.objects.get(pk=self.pk)
+            existing = User.objects.get(pk=self.pk)
+
         except User.DoesNotExist:
             cluster.User(self).create()
+            if self.is_superuser:
+                slack.notify_team(
+                    slack.CREATE_SUPERUSER_MESSAGE.format(username=self.username),
```

Comment:
these "constants" from the other module are probably overkill, IMHO.

Reply:
see new comment on potentially move this formatting inside the `slack` module if you want to enforce the kind of notifications sent and keep them all in the same place.